src/ensembl/package.json

```diff
@@ -43,7 +43,7 @@
     "cross-fetch": "3.1.4",
     "d3": "6.7.0",
     "dotenv": "10.0.0",
-    "ensembl-genome-browser": "https://raw.githubusercontent.com/Ensembl/ensembl-genome-browser-assets/master/assets-80f51620ed443c640cdfd6b5aebd505b.tar.gz",
+    "ensembl-genome-browser": "https://github.com/Ensembl/ensembl-genome-browser#manoj",
```

Comment:
I can't believe this change has been responsible for the deletion of over 42 thousand lines of code in the package-lock file. What if you copy the `package-lock` file from the dev branch, delete your node_modules folder, and run `npm install` again?